crud-framework-core/src/main/java/dev/krud/crudframework/crud/handler/CrudUpdateHandlerImpl.java

```diff
@@ -151,5 +151,64 @@ public <ID extends Serializable, Entity extends BaseCrudEntity<ID>> Entity updat
 		return entity;
 	}
 
+    @Override
+    public <ID extends Serializable, Entity extends BaseCrudEntity<ID>> List<Entity> bulkUpdate(List<Entity> entities, HooksDTO<CRUDPreUpdateHook<ID, Entity>, CRUDOnUpdateHook<ID, Entity>, CRUDPostUpdateHook<ID, Entity>> hooks, boolean applyPolicies) {
+        if (entities.isEmpty()) {
+            throw new IllegalStateException("Cannot update an empty list of entities");
+        }
+
+        Class<Entity> entityClazz = (Class<Entity>) entities.get(0).getClass();
+        List<FieldChangeHook> fieldChangeHooks = crudHelper.getFieldChangeHooks(entityClazz);
+        DynamicModelFilter filter = new DynamicModelFilter()
+                .add(FilterFields.in("id", FilterFieldDataType.get(entities.get(0).getId().getClass()), entities.stream().map(BaseCrudEntity::getId).toArray()));
```

Comment:
May need to box the type, see `CrudHelperImpl#getDataTypeFromClass`

Reply:
Moved the private into the enum, solves the TODO there as well